Sources/PathKit.swift

```diff
@@ -258,15 +248,7 @@ extension Path {
   ///   could not be determined
   ///
   public var isFile: Bool {
-    var directory = ObjCBool(false)
-    guard Path.fileManager.fileExists(atPath: normalize().path, isDirectory: &directory) else {
-      return false
-    }
-#if os(Linux)
-  return !directory
-#else
-  return !directory.boolValue
-#endif
+    return !isDirectory
```

Comment:
Is this always true? What if there's something that is not a directory, but not a file either? (no idea if this is even possible, or if we want to take it into account)